modules/highgui/src/window_w32.cpp

```diff
@@ -1981,6 +1981,97 @@ static void showSaveDialog(CvWindow* window)
     }
 }
 
+/*
+ * message received. check if it belongs to our windows (frame, hwnd).
+ * returns true (and value in result) if there was a keycode.
+ * otherwise return false (keep getting messages).
+ */
+static bool handleMessage(MSG& message, int& result)
```

Comment:
> result

`keyCode`

Reply:
I'll rename it and the corresponding instances in waitKey and pollKey